package.json

```diff
@@ -11,37 +11,20 @@
     "notification",
     "toast"
   ],
-  "main": "dist/react-native-notification.js",
-  "jsnext:main": "dist/react-native-notification.es2015.js",
+  "main": "src/index.js",
   "scripts": {
-    "prebuild": "rimraf dist",
-    "build": "rollup-babel-lib-bundler -f cjs,es6 src/index.js",
-    "prepublish": "in-publish && npm run build || not-in-publish",
     "lint": "eslint src",
     "precommit": "npm run lint",
     "test": "ava --verbose 'src/**/*.test.js'",
     "coverage": "nyc --reporter lcov --reporter html ava 'src/**/*.test.js'"
   },
   "files": [
-    "dist",
```

Comment:
If you have `files` in your `package.json`, you don't need to explicitly set a `.npmignore` file. With `files` in place, the published npm package will **only** contain everything that's defined in this array. (In this case `.npmignore` shouldn't take effect.)